### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by addod19**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+}
+.container{
+    max-width:1440px;
+    margin: 0 auto;
+}
+#my{
+    height: 540px;
+    display: flex;
+    flex-direction: column;
+    background-image: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
```
Comment (Torres-ssf) [inline assets/css/style.css]: Your background image is little off from the right position. In order to fix that, please replace the `background-image` with this one: `background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 0 100%;`

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
Hi there @addod19. You did a great job in this project. But before I give 👍 we need to do some changes to match Microverse guidelines.
- [ ] The most important content in your page must be inside a `main` tag. Your `div id="my"` is a good candidate to be a `main` tag. Kindly change this div tag to a main tag instead.

- [ ] Your main background image is a bit off from the right position. I reviewed your code and suggested a change to fix that.

- [ ] For this project sake, the 3 main sentences must be images. In the original page, those 3 main sentences are actually images. Those images must be loaded in a div tag, using the background, just like you did in the main big image. You can get the 3 image sentences in the original page. Also, all the 3 sentences have to be links. Use a div tags to load the images (no need for the img tag) an put inside an anchor tags.

- [ ] In this project the `footer` starts at your `div class="myfooter"`. Divide the footer with two containers.

- [ ] The four promo images from the first part of the footer must be loaded the same way of the 3 main sentences. Use a div tag for the image and load them using the background. Each one of these images has to be links

Please kindly make these changes and submit for code review again!

**Review by bellom — APPROVED** [review]
Hi @addod19, 
## Great job, Congratulation :+1: 
_It will be great to practice the right way of making using of nav, main and footer tag._
In your code i could see you have `main and footer tag inside div`, it shouldn't be like that, kindly check out this [link](https://www.themelocation.com/best-html5-practices) and correct it.
After the corrction you can merge to master and submit your project completion form.

**Comment by addod19** [comment]
Okay....I will do just that and submit the project completion form as
requested.Thanks

**Comment by bellom** [comment]
:+1:

**Pull request merged**